src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Program.cs

```diff
@@ -90,9 +95,9 @@ private static IHostBuilder EnableLogging(UpgradeOptions options, IHostBuilder h
         }
 
         private static Task<int> RunCommandAsync(
-            UpgradeOptions options,
-            Func<IHostBuilder, IHostBuilder> configure,
-            CancellationToken token)
+    UpgradeOptions options,
```

Comment:
nit: can you revert this?

Reply:
looks like `RunAnalysisAsync` also got formatted left a bit more than intended.

Reply:
fixed the formatting here